plugin/src/gatsby-node.ts

```diff
@@ -280,6 +275,10 @@ exports.createSchemaCustomization = ({
       variants: [ShopifyProductVariant]
     }
 
+    type ShopifyProductFeaturedImage {
+      localFile: File @link
```

Comment:
This is perhaps one thing I'd like to improve. Leave this out of the schema if `downloadImages` is not set.